tools/release.rb

```diff
@@ -101,8 +101,12 @@ def self.generate_release_version(release_type)
 
   private
 
-  def remove_old_releases
-    FileUtils.rm Dir.glob(File.join(Machinery::ROOT, "package/*"))
+  def remove_old_releases(skip_cleanup)
```

Comment:
Make this a named parameter. `remove_old_releases(true)` is much less expressive than `remove_old_releases(skip_cleanup: true)`.